README.md

```diff
@@ -1,8 +1,8 @@
 Microsoft Authentication Library for iOS and macOS
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->